tests/plugins/pastefromword/generated/_fixtures/Enter_BR/word2016/firefox.html

```diff
@@ -0,0 +1,4 @@
+<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:normal'>Paste from word<br>
```

Comment:
I'm not sure if it's correct fixture for Firefox. It should like like the one from Chrome, containing all HTML as Firefox also supports fetching HTML from native `dataTransfer`.